addresssubmission/src/main/java/com/afrigis/services/addresssubmission/AddressSubmissionResult.java

```diff
@@ -0,0 +1,13 @@
+
+package com.afrigis.services.addresssubmission;
+
+/**
+ *
+ * @author Pieterv
+ */
+public interface AddressSubmissionResult {
```

Comment:
Opps, there is absolutely no difference. Was a happy type day. Will should be deleted
@demaniak all above done.